ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -45,6 +45,51 @@
 {%             set framed_mtu = dot1x.radius_av_pair.framed_mtu | arista.avd.default('-') %}
 | {{ radius_av_pair_service }} | {{ framed_mtu }} |
 {%         endif %}
+{%         if dot1x.captive_portal is arista.avd.defined %}
+
+#### 802.1X Captive-portal authentication
+
+| Authentication Attribute | Value |
+| ------------------------ | ----- |
+{%             if dot1x.captive_portal.enabled is arista.avd.defined(true) %}
+| Captive-portal | Enabled |
+{%             endif %}
+{%             if dot1x.captive_portal.url is arista.avd.defined %}
+| URL | {{ dot1x.captive_portal.url }} |
+{%             endif %}
+{%             if dot1x.captive_portal.ssl_profile is arista.avd.defined %}
+| SSL profile | {{ dot1x.captive_portal.ssl_profile }} |
+{%             endif %}
+{%             if dot1x.captive_portal.access_list_ipv4 is arista.avd.defined %}
+| Access-list IPv4 | {{ dot1x.captive_portal.access_list_ipv4 }} |
+{%             endif %}
+{%             if dot1x.captive_portal.start_limit_infinite is arista.avd.defined(true) %}
+| Start limit | Infinite |
+{%             endif %}
+{%         endif %}
+{%         if dot1x.supplicant is arista.avd.defined %}
+
+#### 802.1X Dot1x supplicant
+
+| Attribute | Value |
+| --------- | ----- |
+{%             if dot1x.supplicant.logging is arista.avd.defined(true) %}
```

Comment:
in case supplicant is defined but logging is set as false then we may print only column headings. can you please check

Reply:
Fixed